packages/grafana-ui/src/components/Forms/Checkbox.story.tsx

```diff
@@ -14,36 +14,40 @@ const meta: ComponentMeta<typeof Checkbox> = {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['value', 'htmlValue'],
+    },
   },
 };
 
-export const Controlled = () => {
+export const Controlled: ComponentStory<typeof Checkbox> = (args) => {
   const [checked, setChecked] = useState(false);
   const onChange = useCallback((e) => setChecked(e.currentTarget.checked), [setChecked]);
   return (
     <div>
-      <Checkbox
-        value={checked}
-        onChange={onChange}
-        label="Skip TLS cert validation"
-        description="Set to true if you want to skip TLS cert validation"
-      />
+      <Checkbox value={checked} onChange={onChange} {...args} />
     </div>
   );
 };
 
-export const uncontrolled = () => {
+Controlled.args = {
+  label: 'Skip TLS cert validation',
+  description: 'Set to true if you want to skip TLS cert validation',
+};
```

Comment:
Could we also add `disabled: false` here (and in the `InAField` story), this will mean it will show up in the controls so we can test it as well 😄